app/models/spin.rb

```diff
@@ -127,7 +127,18 @@ def spin_log(log)
   # A boolean representing if the spin is validated or not
   #
   def validate_spin?
-    validate_readme? && validate_metadata?
+    validate_readme? && validate_metadata? && validate_releases?
+  end
+
+  # Validate release
+  #
+  # == Returns:
+  # A boolean representing if the spin has releases
+  #
+  def validate_releases?
+    (return true) unless releases.empty?
+    spin_log("Error in releases, you need  a release in your spin, if you have one refresh the spin")
```

Comment:
"[ERROR] No release found in GitHub. We need a release in the spin so it can be downloaded. Please refresh the spin if you have added one"

Reply:
We could even include some colours...

Reply:
Colours??? In an API ?? This things should be in the front

Reply:
Then the ERROR || WARNING || OK should do